test/unit/Editing.test.ts

```diff
@@ -0,0 +1,33 @@
+import './editing-element.js';
+import { EditingElement } from './editing-element.js';
+import { newActionEvent, newOpenDocEvent } from '../../src/foundation.js';
+import { html, fixture, expect } from '@open-wc/testing';
```

Comment:
In `open-scd` we had an order in imports also in test to make those more readable. This order was something like
1. library imports as line 4 or other stuff like fast check
2. mocks
3. own imports the closer related to the test the lower in the order

I don't know how to formulate something like this, but I like it as it allowed to quickly see missing imports :)